docs(messaging): emphasize iOS simulators do not handle FCM (#4102)
invertase/react-native-firebase: docs/messaging/usage/index.md

## Receiving messages

> FCM messages can be sent to *real* devices (iOS simulators do *not* handle cloud messages, in particular) via a number of methods (see below).

Reviewer: ```suggestion
FCM messages can be sent to *real* Android/iOS devices and Android emulators (iOS simulators however do *not* handle cloud messages) via a number of methods (see below).
```

Reviewer: Thoughts on tweaking slightly? 😅

Author: Yes, that is more precise. Word-smithery always goes better with a bit of back and forth, I value it
